lib/constants.go

```diff
@@ -334,6 +335,9 @@ type EncoderMigrationHeights struct {
 
 	// DeSoAssociations coincides with the AssociationsBlockHeight block
 	DeSoAssociations MigrationHeight
+
+	// DeSoFrozenPosts coincides with the AllowUpdatingNFTPostsBlockHeight block
+	DeSoFrozenPosts MigrationHeight
```

Comment:
ultimately we'll combine all these fork heights into one. Having separate attributes of the ForkHeight struct will make encoder migrations take longer as it appears we have MORE migrations to run. this can be handled later though in a separate PR. we'll come up with some name for the fork